pages/doc/logging_faq.md

```diff
@@ -79,9 +79,9 @@ The Wavefront proxy drops the logs that exceed the [maximum character limit](log
 
 ### See the Data on the Predefined Logs Stats Charts
 
-1. Log in to your Wavefront instance and select **Dashboards** > **All Dashboards**.
-1. Search for **Wavefront Service and Proxy Data** in the search bar, and click on the dashboard.
-    ![A screenshot of the results you get when you search for  the Wavefront Service and Proxy Data dashboard. ](images/logs_wavefront_service_and_proxy_data.png)
+1. On the toolbar, select **Dashboards** > **All Dashboards**.
+1. Search for **Operations for Applications Service and Proxy Data** in the search bar, and click on the dashboard.
```

Comment:
Search for **Operations for Applications Service and Proxy Data** in the search bar, and click the dashboard.